src/lib/tasfw-core/include/tasfw/ScriptStatus.hpp

```diff
@@ -88,7 +88,7 @@ class CompareStatus
 		const AdhocScriptStatus<TCompareStatus>& a,
 		const AdhocScriptStatus<TCompareStatus>& b) const = 0;
 
-	virtual bool Terminator(const AdhocScriptStatus<TCompareStatus>& status) const { return false; }
+	virtual bool Terminator([[maybe_unused]] const AdhocScriptStatus<TCompareStatus>& status) const { return false; }
```

Comment:
I plan to remove the old compare methods after the new ones are finished, but we can put this tag in for now